py3status/modules/group.py

```diff
@@ -49,8 +49,11 @@
         `{button}` not in format (default True)
 
 Format placeholders:
-    {button} The button to open/close or change the displayed group
-    {output} Output of current active module
+    {button} mouse button to toggle or scroll
+    {module} module name
+    {module_full} user/group inputted name for this module
+    {module_user} user inputted name for this module
+    {output} module output
```

Comment:
I think this is clearer

```
    {button} button to open/close or change the displayed module
    {module} module name e.g. `github my_project`
    {module_internal} internally used name for the module
    {module_type} type of module e.g. `github`
    {output} module output
```

Reply:
```
{module_full} ----> {module_internal}
{module}      ----> {module_type}
{module_user} ----> {module}
```
If that's correct, then this may be confusing for many. For one, I think `{module_type}` may be misleading as the users might like to look for names.

Fine with `{module_internal}`. User-inputted name counts as internal too?
I think `{module}` example to include both... might not be used much / confusing. Maybe `{module_id}` --> `my_project` ? eg `{module} {module_id}: {output}` ?

Reply:
I think it is simple `{module}` is the name I gave it in my config so that is what it is called.  `{module_type}` also seems to make sense, because that is the kind of module it is.

`{module_internal}` is a silly thing but could be helpful in bug tracing so I think is good to have.

The important thing is from the users perspective not developers

Reply:
>The important thing is from the users perspective not developers

I'm trying to work out a user perspective. I put in `{name}`, I expect `github` and not `github my_project`. It does not make sense for users to use `{module_type}`... Users should have something rough like...
* `{module} {name}: {output}` // `{module} {fullname}: {output}`
* `{module} {name}: {output}` // `{module} {internal_name}: {output}`
* `{module} {name}: {output}` // `{module} {name_full}: {output}`

if no given name, then nothing shows up.

Reply:
users will just use `{module}` I'm pretty sure.  I think you are over-complicating this.

Reply:
>you are over-complicating this.

Possibly. In your example...
```
{module} module name e.g. `github my_project`
```
Even with `my_project` ? That's what I want to know. Should `my_project` be there? I might not want it there. The users could feel the same. I'll go ahead and make the changes. `{module_type}` just sounds funny right now.

Reply:
Ping.

Reply:
I usually ping people after a few days not 11 minutes ;p

I'm trying to get through the backlog

Anyhow maybe we should wait on this while we ponder things

Reply:
```
    {button} button to open/close or change the displayed module
    {module} module name or type, eg github
    {optional} module's optionally given name, eg py3status
    {internal} module's internally used name, eg _anon_module_0
    {output} module's output, eg (new output here)
```
?